src/elements/content-sidebar/__tests__/useSidebarMetadataFetcher.test.tsx

```diff
@@ -134,41 +142,73 @@ describe('useSidebarMetadataFetcher', () => {
         );
     });
 
-    test('should handle metadata instance removal', async () => {
+    test('should handle metadata instance removal error', async () => {
         mockAPI.getMetadata.mockImplementation((file, successCallback) => {
             successCallback({ templateInstances: mockTemplateInstances, templates: mockTemplates });
         });
-        mockAPI.deleteMetadata.mockImplementation((file, template, successCallback) => {
-            successCallback(mockTemplateInstances[0]);
+        mockAPI.deleteMetadata.mockImplementation((file, template, successCallback, errorCallback) => {
+            errorCallback(mockError, 'metadata_remove_error');
         });
 
         const { result } = setupHook();
-        expect(result.current.templateInstances).toEqual(mockTemplateInstances);
+        expect(result.current.status).toEqual(STATUS.SUCCESS);
```

Comment:
I don't quite understand what happens here. Before doing a call the status is already in `SUCCESS`?

Reply:
It got messed up after the rebase with master 😓 Should be better now